.github/workflows/ci.yml

```diff
@@ -20,6 +20,32 @@ env:
   OCAMLRUNPARAM: b
 
 jobs:
+  test-rewatch-integration:
```

Comment:
This is a real integration test of both rewatch + bsc packaged up in the test repo, to make sure it works end to end